### ISSUE TITLE
Comments are deleted

### ISSUE THREAD

**Issue by andyxmas** [Shopify/prettier-plugin-liquid#144]
**Describe the bug**
The contents between `{% comment %}` and `{% endcomment %}` tags are deleted.

**Unformatted source**
```liquid
{% comment %} 
This is a comment
{% endcomment %}

<p>hello, world!</p>
```

**Expected output**
```liquid
{% comment %} 
This is a comment
{% endcomment %}

<p>hello, world!</p>

```

**Actual output**
```liquid
{% comment %}


{% endcomment %}

<p>hello, world!</p>

```

**Debugging information**
 - OS  Mac OSX 12.5.1
 - Version 1.0.0
 - Node version v18.10.0

**Additional context**
I find this issue when running from the command line like `npx prettier sections/test.liquid --write` or from the plugin on vscode.
I've deleted my .prettier.congig.js

Thanks for you work on this plugin - I'm really keen to adopt it in our workflow, but I've never tested beyond this issue as its a definite blocker.

**Comment by andyxmas** [comment]
@charlespwd ah, you're right. I've set `"editor.renderWhitespace": "all"` in my vscode settings, and I see it now. I shall delete the white space and continue testing!

**Comment by charlespwd** [comment]
Shipped in [v1.0.1](https://github.com/Shopify/prettier-plugin-liquid/releases/tag/v1.0.1), thanks again for reporting :)
